### ISSUE TITLE
Public Gateway add IPv6

### ISSUE THREAD

**Issue by clementd64** [ovh/public-cloud-roadmap#327]
Creating a VM inside a Private Network with a Public Gateway don't assign a IPv6.

Probably related with ovh/infrastructure-roadmap#115

How can you release a new product in 2022 without IPv6 support ?

**Comment by titanshift** [comment]
yes this is a long waited feature thank you @clementd64  for adding this to the roadmap.

**Comment by clementd64** [comment]
When contacting support on 2022-12-08, their said that the Openstack team has no plan on implementing this.
I don't know if thing have changed now.

I think the main problem is that their no commercial profit about implementing it. ARCEP was really clever to use 5G as a carrot to force IPv6 support on french mobile networks.

Selling hosting services without IPv6 is like selling a phone without 4G. As of 2023, it's just stupid.

The only reason we can't implement IPv6 on your sites is simply because a very small detail. No IPFO.
I may eventually accept IPv4-only for internal communication. But everything that public facing must support IPv6, including egress, and with the same or better options that IPv4.

**Comment by matmicro** [comment]
ETA ?

**Comment by clementd64** [comment]
Hello, this is no longer needed as you can attach public IPv6 on the vrack
https://us.ovhcloud.com/network/additional-ip/
https://docs.ovhcloud.com/en/guides/public-cloud/network-services/configure-ip-block-vrack-instance (for IPv4 but tested with IPv6)

Using additional IP (for VRRP or routing subnet for containers) will need allowing there on the anti-spoofing in horizon
